internal/config.go

```diff
@@ -0,0 +1,7 @@
+/*
+All constants or configuartions variables,functions can be declared here
+*/
+
+package internal
+
+var Spatial = []string{"geometrycollection", "multipoint", "multilinestring", "multipolygon", "point", "linestring", "polygon", "geometry"}
```

Comment:
Where is `Spatial` used? Are these types related to `mysql`? I am not sure if this is a typical way how Go organizes configurations, do you have any reference?

Reply:
https://github.com/cloudspannerecosystem/harbourbridge/blob/cd92a11f75fe1688908a6ccc21222ba7861abf3a/mysql/mysqldump.go#L398-L407

Reply:
Because this global variable is only related to mysql. I think it would better to put into `mysqldump.go` at the moment. And please have a more specific variable name, e.g., `MysqlSpatialColumns`.

Reply:
Done. Removed config.go, renamed `Spatial` to `MysqlSpatialDataTypes` and put it in `mysqldump.go`